src/main/java/hudson/remoting/UserRequest.java

```diff
@@ -86,10 +86,32 @@ public UserRequest(Channel local, Callable<?,EXC> c) throws IOException {
         return result;
     }
 
+    static boolean workaroundDone = false;
     protected UserResponse<RSP,EXC> perform(Channel channel) throws EXC {
         try {
             ClassLoader cl = channel.importedClassLoaders.get(classLoaderProxy);
 
+            // Allow forcibly load of a class, allows to workaround:
+            // @See        https://issues.jenkins-ci.org/browse/JENKINS-19445
+            // @Related    https://issues.tmatesoft.com/issue/SGT-451
+            final String clazz = System.getProperty(RemoteClassLoader.class.getName() + ".force", null);
+            if ( clazz != null && !workaroundDone) {
+                // java.lang classes can only be instantiated by the bootstrap Classloader.
+                // Guarantees that *all* threads with whatever Classloader in use, have the
+                // same mutex instance:    an intance of java.lang.Class<java.lang.Object>
+                synchronized(java.lang.Object.class)
+                {
+                    workaroundDone = true;
+                    try {
+                        final Class<?> loaded = Class.forName( clazz, true, cl );
+                        System.err.println("Loaded class: '" + clazz + "' using classloader: " + cl);
```

Comment:
1) Should be java.util.logging.Logger instead of the raw output
2) It would be also great to move such IO operation outside the Object class lock